import-scripts/import-hgnc-data.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+FLOCK_FILEPATH="/data/portal-cron/cron-lock/import-cmo-data-hgnc.lock"
+(
+    # check lock so that script executions do not overlap
+    if ! flock --nonblock --exclusive $flock_fd ; then
+        exit 0
+    fi
+
+    # set necessary env variables with automation-environment.sh
+    if [ -z "$PORTAL_HOME" ] || [ -z "$START_HGNC_IMPORT_TRIGGER_FILENAME" ] || [ -z "$KILL_HGNC_IMPORT_TRIGGER_FILENAME" ] || [ -z "$HGNC_IMPORT_IN_PROGRESS_FILENAME" ] || [ -z "$HGNC_IMPORT_KILLING_FILENAME" ] ; then
+        echo "Error : import-hgnc-data.sh cannot be run without setting the PORTAL_HOME, START_HGNC_IMPORT_TRIGGER_FILENAME, KILL_HGNC_IMPORT_TRIGGER_FILENAME, HGNC_IMPORT_IN_PROGRESS_FILENAME, HGNC_IMPORT_KILLING_FILENAME environment variables. (Use automation-environment.sh)"
+        exit 1
+    fi
+
+    if [ -f "$KILL_HGNC_IMPORT_TRIGGER_FILENAME" ] || [ -f "$HGNC_IMPORT_KILLING_FILENAME" ] ; then
+        # if import kill is happening or is triggered, cancel any import trigger and exit
+        rm -f "$START_HGNC_IMPORT_TRIGGER_FILENAME"
+        exit 1
+    fi
+
+    if ! [ -f "$START_HGNC_IMPORT_TRIGGER_FILENAME" ] ; then
+        # no start trigger for import, so exit
+        exit 1
+    fi
+
+    echo $(date)
+
+    # set data source env variables
+    source $PORTAL_HOME/scripts/set-data-source-environment-vars.sh
+    source $PORTAL_HOME/scripts/clear-persistence-cache-shell-functions.sh
+
+    tmp=$PORTAL_HOME/tmp/import-cron-hgnc
+    if ! [ -d "$tmp" ] ; then
+        if ! mkdir -p "$tmp" ; then
+            echo "Error : could not create tmp directory '$tmp'" >&2
+            exit 1
+        fi
+    fi
+    if [[ -d "$tmp" && "$tmp" != "/" ]]; then
+        rm -rf "$tmp"/*
+    fi
+    now=$(date "+%Y-%m-%d-%H-%M-%S")
+    IMPORTER_JAR_FILENAME="$PORTAL_HOME/lib/hgnc-importer.jar"
+    java_debug_args=""
+    ENABLE_DEBUGGING=0
+    if [ $ENABLE_DEBUGGING != "0" ] ; then
+        java_debug_args="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=27183"
+    fi
+    JAVA_IMPORTER_ARGS="$JAVA_PROXY_ARGS $java_debug_args $JAVA_SSL_ARGS -Dspring.profiles.active=dbcp -Djava.io.tmpdir=$tmp -ea -cp $IMPORTER_JAR_FILENAME org.mskcc.cbio.importer.Admin"
+    hgnc_notification_file=$(mktemp $tmp/hgnc-portal-update-notification.$now.XXXXXX)
+    ONCOTREE_VERSION_TO_USE=oncotree_candidate_release
+    DATA_SOURCES_TO_BE_FETCHED="bic-mskcc cmo-argos private impact impact-MERGED knowledge-systems-curated-studies immunotherapy datahub datahub_shahlab msk-mind-datahub pipelines-testing"
+    unset failed_data_source_fetches
+    declare -a failed_data_source_fetches
+
+    # import is beginning - create status file showing "in_progress", and remove trigger
+    rm -f "$START_HGNC_IMPORT_TRIGGER_FILENAME"
+    touch "$HGNC_IMPORT_IN_PROGRESS_FILENAME"
+
+    CDD_ONCOTREE_RECACHE_FAIL=0
+    if ! [ -z $INHIBIT_RECACHING_FROM_TOPBRAID ] ; then
```

Comment:
where is this set? 
Also let's the check off for now since CDD can't refresh due to the missing MSKVocab project